docs/ref/applications.txt

```diff
@@ -431,6 +436,11 @@ application registry.
     It must be called explicitly in other cases, for instance in plain Python
     scripts.
 
+    .. versionchanged:: 5.0
+
+        Raises a ``RuntimeWarning`` when apps interact with the database before
+        the app registry has been fully populated.
```

Comment:
I don't think this note is needed, it's duplicative?

Reply:
Yes, but the two notes are somewhat far apart and are attached to two different methods. If I _had_ to choose one of the two instances, I would keep the one below `django.setup()` because `django.setup()` actually triggers the warning. The one below `AppConfig.ready()` just felt natural because it was right below the big fat warning block about execute SQL queries in `AppConfig.ready()`.

Reply:
I've removed the note blow `AppConfig.ready()` and kept the one below `django.setup()`